### ISSUE TITLE
Character glitch in house

### ISSUE THREAD

**Issue by xZedys** [fkk-cz/noire-public#796]
I recently got a new house one in the hills which i've been wanting but for some reason in two areas of the house my character likes to disappear and reappear super fast. Is there any fix to this? Really not wanting to have to switch houses thank you.

**Comment by FertileSphere** [comment]
you can't fix this sadly, you just have to deal with some void space.

**Comment by Bountyrogger** [comment]
Yeah my old property had this issue on the middle flor. One of the main reasons I moved
